sound/pci/hda/patch_hdmi.c

```diff
@@ -2194,7 +2194,9 @@ static int generic_hdmi_build_controls(struct hda_codec *codec)
 
 	for (pin_idx = 0; pin_idx < spec->num_pins; pin_idx++) {
 		struct hdmi_spec_per_pin *per_pin = get_pin(spec, pin_idx);
+		struct hdmi_eld *pin_eld = &per_pin->sink_eld;
 
+		pin_eld->eld_valid = false;
```

Comment:
typo in commit message:

 at end the of its probe,

and this can be delayed by e.g. the initial DSP load -> and the workqueue-based probe?

Due to this delay, DRM driver may already send the initial monitor ELD data via acomp.

may already have sent?